css/style.css

```diff
@@ -1 +1,5 @@
-/* file for your css */
\ No newline at end of file
+/* file for your css */
+
+.wrong {
+  border: 1px solid red;
+}
```

Comment:
Here is my CSS for red and green on win or fail:

    .has-success {
        background-color: green;
    }

    .has-fail {
        background-color: red;
    }